apps/etterlatte-vedtaksvurdering/src/test/kotlin/no/nav/etterlatte/vedtaksvurdering/database/DatabaseExtension.kt

```diff
@@ -0,0 +1,35 @@
+package no.nav.etterlatte.vedtaksvurdering.database
+
+import no.nav.etterlatte.GenerellDatabaseExtension
+import org.junit.jupiter.api.extension.AfterAllCallback
+import org.junit.jupiter.api.extension.BeforeAllCallback
+import org.junit.jupiter.api.extension.ExtensionContext
+
+/**
+ * Det tar veldig mye tid å kjøre opp stadig nye Postgres-containere og kjøre Flyway migreringer.
+ * Denne extensionen kjører opp èn instans, som så gjenbrukes av de som måtte ønske det.
+ */
+object DatabaseExtension : BeforeAllCallback, AfterAllCallback, ExtensionContext.Store.CloseableResource {
```

Comment:
Kunne du ikke bare gjort likt som denne? 

https://github.com/navikt/pensjon-etterlatte-saksbehandling/blob/main/apps/etterlatte-behandling/src/test/kotlin/DatabaseExtension.kt

Reply:
Må uansett tilpasse meg til den med nyeste fra main, ja. Så det litt for seint